apiserver/facades/agent/instancemutater/instancemutater.go

```diff
@@ -51,13 +47,22 @@ type InstanceMutaterAPI struct {
 	*common.LifeGetter
 
 	st          InstanceMutaterState
-	model       ModelCache
+	watcher     InstanceMutatorWatcher
 	resources   facade.Resources
 	authorizer  facade.Authorizer
 	getAuthFunc common.GetAuthFunc
 	machineFunc EntityMachineFunc
 }
 
+type InstanceMutatorWatcher interface {
+	WatchLXDProfileVerificationForMachine(id string) (*MachineLXDProfileWatcher, error)
+}
+
+type instanceMutatorWatcher struct {
+	st      InstanceMutaterState
+	metrics *cache.ControllerGauges
```

Comment:
I say remove all the metrics, including `LXDProfileChangeError`, `LXDProfileChangeNotification` and `LXDProfileNoChange` from _core/cache/metrics.go_ and look at adding them later.